### ISSUE TITLE
build es6 version of the sql-formatter

### ISSUE THREAD

**Issue by nassimbg** [sql-formatter-org/sql-formatter#105]
In order to use sql-formatter in an es6 project I had to import it in this format `import sqlFormatter from 'sql-formatter/src/sqlFormatter.js';` and then download all the dependencies for sql-formatter which is currently only "lodash" in its es6 version.

I would like to use an es6 version of the sql-formatter at runtime without any bundling/packaging step so that we can import it in a proper es6 format import `import sqlFormatter from "sql-formatter";`

**Comment by JounQin** [comment]
@nene Any news? The source code is written in ES module, so the simplest way is just adding a `module` field into `package.json`: `"module": "src/sqlFormatter.js"`.

---

Workaround for now is using `alias` feature of webpack in our own codes.

**Comment by inferrinizzard** [comment]
Hi, this issue is fixed in my fork here:
https://github.com/inferrinizzard/prettier-sql
https://www.npmjs.com/package/prettier-sql

**Comment by nene** [comment]
This should now be finally fixed in 10.8.0-beta.1

**Comment by JounQin** [comment]
@nene 

I just take a look https://unpkg.com/browse/sql-formatter@10.8.0-beta.1/package.json

It seems only `.` with a native ESM entry without `"type": "module"` are included, so it may be wrong and should be:

```diff
"exports": {
-  ".": "./lib/index.js"
+  "types": "./lib/src/index.d.ts",
+  "import": "./lib/index.js",
+  "require": "./dist/sql-formatter.min.js"
},
```

**Comment by nene** [comment]
I don't quite understand. You're saying that there is no `"type": "module"`, but there definitely is at line 15:

```
  "type": "module",
```

**Comment by JounQin** [comment]
> I don't quite understand. You're saying that there is no `"type": "module"`, but there definitely is at line 15:
> 
> ```
>   "type": "module",
> ```

Sorry I miss read, then `.js` is considered as ESM **only**.

`require('sql-formatter')` will throw because there is no `commonjs` entry declared in `exports` field (`main` field is only for legacy node version).

**Comment by nene** [comment]
Aha... I see.

**Comment by nene** [comment]
@JounQin would you mind checking if this looks OK to you: https://github.com/sql-formatter-org/sql-formatter/pull/467/files

**Comment by nene** [comment]
Released 10.8.0-beta.2

Hopefully this one will work better.
